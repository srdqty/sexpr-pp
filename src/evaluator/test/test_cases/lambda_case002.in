(letrec
  ((f (lambda (x y) x y)))
  (f))
