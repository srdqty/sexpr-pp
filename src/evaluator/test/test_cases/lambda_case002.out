eval: function expected 2 arguments, but got 0 arguments
